test/MessageTraitTest.php

```diff
@@ -229,8 +234,9 @@ public function testWithHeaderReplacesDifferentCapitalization()
      */
     public function testWithAddedHeaderRaisesExceptionForNonStringNonArrayValue($value)
     {
-        $this->setExpectedException('InvalidArgumentException', 'must be a string');
-        $message = $this->message->withAddedHeader('X-Foo', $value);
+        $this->expectException(InvalidArgumentException::class, 'must be a string');
```

Comment:
Same comment here, and throughout; any two-argument calls to `setExpectedException()` need to be re-written as two discrete calls to `expectException()` and `expectExceptionMessage()`.